package.json

```diff
@@ -694,6 +694,10 @@
             "description": "Disable any queued tasks while running the task. This includes the auto resolve when Packages.resolved is updated.",
             "type": "boolean"
           },
+          "dontTriggerTestDiscovery": {
```

Comment:
Suggestion: name positively "triggerTestDiscovery", to avoid a double negation when assigning `false`.

Reply:
The default should be trigger test discovery, so I'm not keen on changing this